framework/core/device.cpp

```diff
@@ -40,7 +40,27 @@ Device::Device(PhysicalDevice &gpu, VkSurfaceKHR surface, std::unordered_map<con
 	{
 		const VkQueueFamilyProperties &queue_family_property = gpu.get_queue_family_properties()[queue_family_index];
 
-		queue_priorities[queue_family_index].resize(queue_family_property.queueCount, 1.0f);
+		if (gpu.has_high_priority_graphics_queue())
+		{
+			uint32_t graphics_queue_family = get_queue_family_index(VK_QUEUE_GRAPHICS_BIT);
+			if (graphics_queue_family == queue_family_index)
+			{
+				queue_priorities[queue_family_index].reserve(queue_family_property.queueCount);
```

Comment:
I think this should be ```resize``` instead of ```reserve```. With the latter, this crashes for my on Windows with MSVC2019.

Reply:
Fixed.